lib/services/deviceprovisioningservicesManagement/lib/README.md

```diff
@@ -0,0 +1,35 @@
+# Microsoft Azure SDK for Node.js - IotDpsClient
+This project provides a Node.js package for accessing Azure. Right now it supports:
+- **Node.js version 6.x.x or higher**
+
+## Features
+
+
+## How to Install
+
+```bash
+npm install azure-arm-deviceprovisioningservices
+```
+
+## How to use
+
+### Authentication, client creation and list operations as an example.
+
+```javascript
+import * as msRest from "ms-rest";
+import { IotDpsClient, IotDpsModels } from "azure-arm-deviceprovisioningservices";
+const subscriptionId = "<Subscription_Id>";
+const token = "<access_token>";
+const creds = new msRest.TokenCredentials(token);
```

Comment:
This is not a correct example. We should be showing an example with one of the three methods (`msRestAzure.loginWithServiceSprincipalSecret, .interactiveLogin(), .loginWithUsernamePassword()`

Reply:
Take a look at this example as a template https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/advisorManagement#authentication-client-creation-and-listing-recommendations-as-an-example

Reply:
An example using  callbacks or promises or even the latest async/await is ok (any one of the styles is fine).